src/main/java/TaskManager.java

```diff
@@ -0,0 +1,61 @@
+public class TaskManager {
+
+    private static final String INVALID_COMMAND_MESSAGE = "Invalid command. Please try again.";
+
+    private Task[] tasks = new Task[100];
+    int taskCount = 0;
+
+    public String listTask() {
+        StringBuilder feedback = new StringBuilder();
+        
+        feedback.append("Here are the tasks in your list:" + System.lineSeparator());
+        for (int i=0; i<taskCount-1; ++i) {
+            feedback.append(String.format("%d: %s", (i + 1), tasks[i])).append(System.lineSeparator());
+        }
+        feedback.append(String.format("%d: %s", (taskCount), tasks[taskCount-1]));
+
+        return feedback.toString();
+    }
+
+    public String addTask(String taskType, String description) {
+        switch (taskType) {
+        case "todo":
+            tasks[taskCount] = new Todo(description);
+            break;
+        case "deadline": {
+            String[] nameAndDate = parseDescription(description, " /by ");
+            tasks[taskCount] = new Deadline(nameAndDate[0], nameAndDate[1]);
+            break;
+        }
+        case "event": {
+            String[] nameAndDate = parseDescription(description, " /at ");
+            tasks[taskCount] = new Event(nameAndDate[0], nameAndDate[1]);
+            break;
+        }
+        default:
+            return INVALID_COMMAND_MESSAGE;
+        }
+
+        ++taskCount;
+
+        return "Got it. I've added this task:" + System.lineSeparator()
+                + tasks[taskCount-1] + System.lineSeparator()
+                + "Now you have " + taskCount + " tasks in the list.";
+    }
+
+    private String[] parseDescription(String description, String regex) {
+        final String[] split = description.split(regex);
+        if(split.length == 2){
+            return split;
+        }else{
+            return new String[]{split[0], ""};
+        }
```

Comment:
It might be better to have spaces between the conditional statement and opening braces.